feature-libs/user/package.json

```diff
@@ -22,7 +22,8 @@
     "test:schematics": "npm --prefix ../../projects/schematics/ run clean && npm run clean:schematics && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
   },
   "dependencies": {
-    "tslib": "^2.6.2"
+    "tslib": "^2.6.2",
+    "angular-oauth2-oidc": "^17.0.1"
```

Comment:
why do we need to introduce this lib, should be there already

Reply:
create a ticket to track this  https://jira.tools.sap/browse/CXSPA-8184